addons/web_editor/static/lib/odoo-editor/src/commands/commands.js

```diff
@@ -424,9 +424,9 @@ export const editorCommands = {
         }
     },
     removeFormat: editor => {
-        editor.document.execCommand('removeFormat');
```

Comment:
Removing this is likely to have more effect than the one intended in the PR. The `removeFormat` command is a command on its own. It even has its own button in the toolbar. If I'm not mistaken, removing this line completely breaks the feature, though I'm surprised it doesn't break some tests. Maybe this command is not well tested considering we mostly rely on the native `document.execCommand` ?

Reply:
Ahh indeed, maybe we can add a test for `removeFormat` here? For the fix, I think then it's better to specifically remove the gradient image when switching tags. I did the fix in the removeFormat cause I didn't want the repeated code and the tests were alright after removing it :pray:

Reply:
And if we agree that the styles should be kept after the tag change, then actually we only need to remove the `removeFormat` step in `setTag` :rofl: I think I misunderstood your comment on the first fix :grimacing:

Reply:
@Jinjiu96 Well that is a good question, in fact. I think both keeping and not keeping styles on `setTag` can be argued about. It seems we have been going back and forth on that for a time so I'll ask Simon our new PO on the task.